code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -44,6 +44,13 @@
 		to_chat(user, "<span class='warning'>It's locked!</span>")
 	return
 
+/obj/item/storage/lockbox/AltClick(mob/user)
+	if(Adjacent(user) && allowed(user))
+		locked = !locked
+		to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] \the [src] interface.</span>")
+		icon_state = "[locked ? icon_locked : icon_closed]"
+	else
+		to_chat(user, "<span class='warning'>Access denied.</span>")
```

Comment:
This will say "Access denied." even if you're not adjacent. Adjacent should probably just return early

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/21987702/3599190e-cc9b-48d1-b9f7-f3bade7e2a1e)
You should not be able to remotely open them with TK.

Reply:
Its not that, its that you can alt click it from anywhere on the screen. It just needs no to_chat sent to the user if theyre not adjacent to it

Reply:
I misunderstood, yeah, i see saw and issue, and added an early return.